data/json/move_modes.json

```diff
@@ -0,0 +1,47 @@
+[
+  {
+    "type": "movement_mode",
+    "id": "walk",
+    "character": "w",
+    "name": "walk",
+    "exertion_level": "LIGHT_EXERCISE",
+    "change_good_none": "You start walking.",
+    "change_good_animal": "You nudge your steed into a steady trot.",
+    "change_good_mech": "You set your mech's leg power to a loping fast walk.",
+    "move_type": "walking",
+    "move_speed_multiplier": 1.0
+  },
+  {
+    "type": "movement_mode",
+    "id": "run",
+    "character": "r",
+    "name": "run",
+    "exertion_level": "ACTIVE_EXERCISE",
+    "change_good_none": "You start running.",
+    "change_good_animal": "You spur your steed into a gallop.",
+    "change_good_mech": "You set the power of your mech's leg servos to maximum.",
+    "change_bad_none": "You're too tired to run.",
+    "change_bad_animal": "Your steed is too tired to go faster.",
+    "change_bad_mech": "Your steed is too tired to go faster.",
```

Comment:
Something about mech's leg servos here?

Reply:
This is what it was previously - if you have any ideas, I'd be happy to change it.

Reply:
This option has not been implemented, AFAICT. But if he will be in the future, what about "Your mech's leg servos are unable to operate faster."?